lib/matplotlib/patches.py

```diff
@@ -1991,17 +1991,14 @@ def __init__(self, pad=0.3):
 
         def transmute(self, x0, y0, width, height, mutation_size):
             pad = mutation_size * self.pad
-
             # width and height with padding added.
-            width, height = width + 2*pad, height + 2*pad
-
+            width = width + 2 * pad
+            height = height + 2 * pad
```

Comment:
Could be:
```suggestion
            width += 2 * pad
            height += 2 * pad
```
but also it's a bit odd to have changed this one in the first place, because there are a whole lot more below unchanged.

Reply:
at least in theory, width and height *could* be numpy 0d arrays which your patch would incorrectly modify in-place.

Reply:
I guess I was a bit more interested in the second part of the comment.

Reply:
fair point, I guess this doesn't buy much, I just put everything back in a single line.